src/model/data_access_layer/groups/groups_redactor.py

```diff
@@ -0,0 +1,17 @@
+from sqlalchemy.orm.query import Query
+
+from src.model.tables import Groups
+
+
+class GroupsRedactor:
+
+    def __init__(self, alchemist):
+        self.session = alchemist.session
+
+    def insert_values(self,
+                      **kwargs: dict) -> Query:
+        record = Groups(
```

Comment:
Looks like this method should create `Group` instance. Please rename it to `create_group`